sshcommand

```diff
@@ -136,6 +136,9 @@ sshcommand-acl-add() {
     KEY=$(cat "$KEY_FILE")
   fi
 
+  local count="$(wc -l <<< "$KEY")"
+  [ "$count" -eq 1 ] || log-fail "Too many keys provided, set one per invocation of sshcommand acl-add <USER> <NAME>"
```

Comment:
Fixed. Was just pulling that straight from dokku's [ssh-keys:add](https://github.com/dokku/dokku/blob/4e6a699140686a4700e821a18fd9c230d42f6ba9/plugins/ssh-keys/subcommands/add#L20-L21).